_cloud/__init__.py

```diff
@@ -9,7 +9,11 @@
 
 
 class Module(object):
-    """The magical API around downloading / installing /Â importing things."""
+    """The magical API around downloading / installing / importing things."""
+    
+    _zip = None
+    metadata = None
```

Comment:
There are lines which define `self._zip` and `self.metadata` out of the `__init__` function (which triggers pylint for some reason).